src/test/java/org/example/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
It's good that you've added the unit test to your class but what is the purpose of it?

Reply:
This file was generated automatically by Maven